tiledb/sm/serialization/query.cc

```diff
@@ -49,6 +49,8 @@ namespace serialization {
 
 #ifdef TILEDB_SERIALIZATION
 
+enum SerializationContext { CLIENT, SERVER, BACKUP };
```

Comment:
Since this is C++ let's change this to `enum class SerializationContext { Client ...`

Reply:
Done